src/MOI/MOI_wrapper.jl

```diff
@@ -369,13 +369,13 @@ function MOI.get(model::Optimizer, ::MOI.ListOfConstraintAttributesSet)
 end
 
 function _indices_and_coefficients(
-    indices::AbstractVector{Cint},
+    indices::AbstractVector{<:Integer},
```

Comment:
I think we should generalize this at the API level, not here. Also, I don't think is good to deviate a lot from CPLEX this early, unless there a big reason for that. We are still learning about the new API, one could learn from the other.

Reply:
The difference is that in the CPLEX interface everything is made to be `Cint` in the MOI wrapper before it is sent to the API. Talking with @kdheepak we thought it would be better to allow Julia using multiple-dispatch to take care of the Integer type.